Paralayout/UIViewAdditions_Sizing.swift

```diff
@@ -120,8 +120,10 @@ public extension UIView {
     
     /// Resize the view to a set width, and unlimited height (e.g. when in a scroll view).
     /// - parameter width: the width to set.
-    public func wrap(toFitWidth width: CGFloat) {
-        resize(toFit: CGSize(width: width, height: .greatestFiniteMagnitude), constraints: .wrap)
+    /// - parameter height: the height to fit (optional, defaults to `greatestFiniteMagnitude`).
+    /// - parameter margins: An inset from the supplied width to use (optional, defaults to `0`).
```

Comment:
Since this takes `width` and `height`, it's unclear from the method name what `margins` applies to.  Maybe we should make this `wrap(toFitWidth:height:horizontalMargins:verticalMargins:)`, or something similar?